fpdf/fpdf.py

```diff
@@ -335,6 +335,7 @@ def __init__(
         self.font_family = ""  # current font family
         # current font style (BOLD/ITALICS - does not handle UNDERLINE nor STRIKETHROUGH):
         self.font_style = ""
+        self.font_size_set_for_page = False  # current font and size are already added to current page contents with _out
```

Comment:
```suggestion
        self._font_size_set_for_page = False  # current font and size are already added to current page contents with _out
```
Let's use a private attibute: no need for end-users to fiddle with that 🙂